antlr_plsql/ast.py

```diff
@@ -3,18 +3,21 @@
 import antlr_ast
 from antlr_ast import AstNode, Speaker
 
-from . import plsql_grammar
+from . import grammar
 
 # AST -------------------------------------------------------------------------
 # TODO: Finish Unary+Binary Expr
 #       sql_script
 
 
-def parse(sql_text, start='sql_script', strict=False):
-    return antlr_ast.parse(plsql_grammar, AstVisitor(), sql_text, start, strict)
+def parse(sql_text, start="sql_script", strict=False):
+    tree = antlr_ast.parse(grammar, sql_text, start, strict)
+    return AstVisitor().visit(tree)
 
 
 import yaml
+
+
 def parse_from_yaml(fname):
     data = yaml.load(open(fname)) if isinstance(fname, str) else fname
```

Comment:
Better to use a context manager to open a file. Also  not sure what the default is for `open`, but you only need to read here so second argument can be `"r"`. Also, I think it's better to use `yaml.safe_load` here as well

Reply:
(if you don't use a context manager, file will remain open throughout the process. This can be limited by the container)